helpers/utils.js

```diff
@@ -47,18 +48,22 @@ const sendSongMessage = (message, gameSession, isForfeit) => {
         }
     })
 }
-const sendInfoMessage = (message, title, description) => {
-    message.channel.send({
-        embed: {
-            color: EMBED_INFO_COLOR,
-            author: {
-                name: message.author.username,
-                icon_url: message.author.avatarURL()
-            },
-            title: `**${title}**`,
-            description: description
+const sendInfoMessage = (message, title, description, footerText, footerImageWithPath, footerImage) => {
```

Comment:
What happens if footer isn't specified?

Reply:
Why does need footer image with path and footer image

Reply:
If they aren't specified nothing happens. They're optional and the embed behaves as it should when they aren't given, without any errors.

Reply:
The reason I need to differentiate with the path and the regular filename is because the embed needs the path to the file when adding an attachment, but when being used in the embed, it needs only the filename. I can actually get away with just passing the path by using something along the lines of:
```js
var path = require('path');
var file = '/home/user/dir/file.txt';

var filename = path.parse(file).base;
```